package.json

```diff
@@ -18,41 +18,42 @@
   "devDependencies": {
     "babel-eslint": "^8.1.2",
     "babel-jest": "^22.0.4",
-    "babel-loader": "^7.1.2",
+    "babel-loader": "^7.1.4",
     "babel-plugin-transform-class-properties": "^6.23.0",
     "babel-plugin-transform-runtime": "^6.15.0",
     "babel-preset-es2015": "^6.18.0",
     "babel-preset-flow": "^6.23.0",
     "babel-register": "^6.26.0",
     "benchmark": "^2.1.4",
     "check-dependencies": "^1.1.0",
-    "css-loader": "^0.28.8",
+    "css-loader": "^0.28.11",
     "eslint": "^4.14.0",
     "eslint-plugin-flowtype": "^2.40.1",
-    "extract-text-webpack-plugin": "^3.0.2",
-    "file-loader": "^1.1.6",
+    "file-loader": "^1.1.11",
     "flow-bin": "^0.62.0",
     "jest": "^22.0.4",
     "jest-serializer-html": "^4.0.1",
     "js-yaml": "^3.10.0",
     "json-stable-stringify": "^1.0.1",
     "jspngopt": "^0.2.0",
     "less": "~2.7.1",
-    "less-loader": "^4.0.5",
+    "less-loader": "^4.1.0",
+    "mini-css-extract-plugin": "^0.4.0",
     "mkdirp": "^0.5.1",
     "pako": "1.0.4",
     "pre-commit": "^1.2.2",
     "query-string": "^5.1.0",
     "rimraf": "^2.6.2",
     "selenium-webdriver": "^2.48.2",
     "sri-toolbox": "^0.2.0",
-    "style-loader": "^0.19.1",
+    "style-loader": "^0.21.0",
```

Comment:
`style-loader` may have issues with webpack 4: https://github.com/webpack-contrib/style-loader/pull/282. But it seems to work.